figma-tokens.json

```diff
@@ -480,6 +522,84 @@
           "value": "{borderRadii.3}",
           "type": "borderRadii"
         }
+      },
+      "sm": {
+        "value": "{borderRadius.md}",
+        "type": "borderRadius"
```

Comment:
Is it possible to rename this borderRadius type to borderRadii? if not we will need to refactor